posthog/temporal/data_imports/pipelines/postgres/postgres.py

```diff
@@ -92,6 +93,25 @@ class TableStructureRow:
     numeric_scale: Optional[int]
 
 
+def _get_partition_bucket_size(cursor: psycopg.Cursor, schema: str, table_name: str) -> int | None:
+    query = sql.SQL("""
+        SELECT round(({bytes_per_partition}) / (pg_total_relation_size({schema_table_name_literal}) / count(*)))
+        FROM {schema}.{table}""").format(
```

Comment:
`pg_total_relation_size` Also includes index size, are we also importing those? Otherwise we could be overestimating here. Why not use `pg_table_size` instead?

Reply:
Didn't know about `pg_table_size` - will use that instead